services/web/client/source/class/osparc/component/form/Auto.js

```diff
@@ -448,12 +452,16 @@ qx.Class.define("osparc.component.form.Auto", {
         Object.assign(s, s.contentSchema);
       }
 
-      if (s.defaultValue) {
-        if (!s.set) {
-          s.set = {};
-        }
+      if (!s.set) {
+        s.set = {};
+      }
+      if ("defaultValue" in s) {
```

Comment:
just a question - would this apply default values to arrays as well? it seems that the integer, number and dropdowns have their own handling of default values but not necessarily arrays - it would be awesome to have an array default!